node/src/rpc.rs

```diff
@@ -63,7 +97,7 @@ where
 	let FullDeps { client, pool, deny_unsafe } = deps;
 
 	module.merge(System::new(client.clone(), pool, deny_unsafe).into_rpc())?;
-	module.merge(TransactionPayment::new(client).into_rpc())?;
+	module.merge(TransactionPayment::new(client.clone()).into_rpc())?;
```

Comment:
We don't need to clone it if this is the last time we use it no?

Reply:
Done.